News/Entities/Models/UserCreateModel.cs

```diff
@@ -2,7 +2,7 @@
 
 public class UserCreateModel
 {
-    public string? Username { get; set; }
+    public string? FirstName { get; set; }
```

Comment:
For creating a user profile, should the user only provide a first name without a last name, or all name should it be in one line?

Reply:
Just first name should be enough I guess.

Reply:
FirtsName enough for now